core/java/com/google/instrumentation/stats/ViewDescriptor.java

```diff
@@ -81,6 +79,43 @@ private ViewDescriptor(
   }
 
   /**
+   * Visitor for the ViewDescriptor type.
```

Comment:
I wanted to make it clear that I applied the visitor pattern to ViewDescriptor.  Do you think that it's already clear enough from the code?